jinahub/encoders/text/DPRTextEncoder/dpr_text.py

```diff
@@ -105,8 +122,23 @@ def encode(self, docs: Optional[DocumentArray], parameters: dict, **kwargs):
             for batch_docs in document_batches_generator:
                 with torch.no_grad():
                     texts = batch_docs.get_attributes('text')
+                    if self.encoder_type == 'context' and self.title_tag_key:
+                        text_pair = batch_docs.get_attributes(
+                            f'tags__{self.title_tag_key}'
+                        )
+                        if len(text_pair) != len(batch_docs):
+                            raise ValueError(
+                                'If you set `title_tag_key` property, all documents'
+                                ' that you want to encode must have this tag. Found'
+                                f' {len(text_pair) - len(batch_docs)} documents'
+                                ' without it.'
+                            )
+                    else:
+                        text_pair = None
+
                     inputs = self.tokenizer(
-                        texts,
+                        text=texts,
+                        text_pair=text_pair,
```

Comment:
I do not see in the documentation what these `text_pair` refer to.

Reply:
This is from one of the the superclasses
https://huggingface.co/transformers/internal/tokenization_utils.html?highlight=text_pair#transformers.tokenization_utils_base.PreTrainedTokenizerBase.__call__